packages/website/src/pages/foundations/Iconography.tsx

```diff
@@ -1,34 +1,101 @@
-        </ul>
-    </PlasmaComponent>
+const {iconsList, ...Icons} = PlasmaReactIcons;
+
+const Sandbox = dynamic(
+    import('../../building-blocs/Sandbox').then((mod) => mod.Sandbox),
+    {ssr: false, loading: () => <PlasmaLoading />}
+);
+
+const TableWithFilter: React.ComponentType<
+    TableWithFilterProps & ITableHOCProps & React.HTMLAttributes<HTMLTableElement>
+> = tableWithFilter()(TableHOC);
+
+type IconSet = {iconName: string; variants: Array<keyof typeof Icons>};
+
+const IconSetCard: React.FunctionComponent<IconSet> = ({iconName, variants}) => (
+    <div key={iconName} className="card p2 flex-column">
+        <h6 className="h6-subdued mb2">{iconName}</h6>
+        <table className="table">
+            <thead className="mod-no-border-top">
+                <tr>
+                    <th>Icon</th>
+                    <th>Name</th>
+                </tr>
+            </thead>
+            <tbody>
+                {variants.map((svgName) => {
+                    const SvgComponent = Icons[svgName];
+                    return SvgComponent ? (
+                        <tr key={svgName}>
+                            <td className="mod-no-border-bottom">
+                                <SvgComponent height={parseInt(/(\d+)px/i.exec(svgName)[0], 10)} />
+                            </td>
+                            <td className="mod-no-border-bottom">{svgName}</td>
+                        </tr>
+                    ) : null;
+                })}
+            </tbody>
+        </table>
+    </div>
 );
 
-const IconItem = ({svgName, fileName}: {svgName: SvgName; fileName: string}) => (
-    <li className="cursor-pointer">
-        <LinkSvg svg={{svgName}} url={getSvgURL(fileName)}>
-            <label className="icon-name cursor-pointer">{svgName}</label>
-        </LinkSvg>
-    </li>
+const IconsTable: React.FunctionComponent = () => (
+    <TableWithFilter
+        id="iconography"
+        data={iconsList}
+        className="plasma-page-layout__section full-content"
+        tbodyClassName="icon-grid"
+        renderBody={(data: typeof iconsList) =>
+            data.map(({iconName, variants}) => (
+                <IconSetCard key={iconName} iconName={iconName} variants={variants as Array<keyof typeof Icons>} />
+            ))
+        }
+        filterBlankslate={{
+            title: <span className="h4-book mb1 nowrap">No icons match the applied filter</span>,
+        }}
+    />
 );
 
-const getSvgURL = (fileName: string) =>
-    `https://github.com/coveo/plasma/blob/master/packages/style/resources/icons/svg/${fileName}`;
+const code = `
+    import * as React from 'react';
+    import { InfoSize32Px } from '@coveord/plasma-react-icons';
+
+    
+    // Control the size using "height" or "width" attributes (defaults to 1em)
+    // The icon takes the same color as the text around it
+
+    export default () => (
+        <div style={{color: 'var(--info-60)'}}>
+            <InfoSize32Px height={64} />;

```

Comment:
```suggestion
            <InfoSize32Px height={64} />

```
I think it's this one :see_no_evil: 
![image](https://user-images.githubusercontent.com/63734941/155713205-37a0338b-9506-4642-8b5b-0162fcdb89da.png)